lighthouse-core/audits/consistently-interactive.js

```diff
@@ -54,7 +54,7 @@ class ConsistentlyInteractiveMetric extends Audit {
    * @return {!Array<!TimePeriod>}
    */
   static _findNetworkQuietPeriods(networkRecords, traceOfTab) {
-    const traceEnd = traceOfTab.timestamps.traceEnd;
+    const traceEndTsInMs = traceOfTab.timestamps.traceEnd / 1000;
```

Comment:
ha, yeah, exactly. I figured that's what this was doing, but then my next thought was, "wait, what is this even used for?"